Assets/Scripts/DialogueManager.cs

```diff
@@ -88,128 +127,131 @@ public void AddSpeaker(SpeakerContainer speaker)
         }
 
         speakers.Add(speaker.id, speaker);
-    }
+    }
 
 
-    public void StartConversation(Conversation convo)
+    public void StartConversation(Conversation conversation)
     {
-        // Stop any autoplay conversation
-        if (coroutine != null)
-        {
-            StopCoroutine(coroutine);
-            coroutine = null;
-        }
-
-        currentConvo = convo;
         startFrame = Time.frameCount;
 
-        if (convo.autoplayConversation)
+        // Get all speakers from the convo
+        List<string> convoSpeakers = conversation.GetAllSpeakers();
+
+        StopAllConversations(convoSpeakers);
+
+        ActiveConversation startedConvo = new ActiveConversation(conversation);
+        activeConversations.Add(conversation, startedConvo);
```

Comment:
```suggestion
        activeConversations.Add(conversation, new ActiveConversation(conversation));
```
since startedConvo doesnt seem to be used after this?